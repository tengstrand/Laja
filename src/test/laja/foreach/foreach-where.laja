
#set (even = [ true, false, true, false, true ])

#write "target/test-output/foreach-where.out"
  #foreach (x in [1,2,3,4] where even[x])
value = {x}
  #end
#end
