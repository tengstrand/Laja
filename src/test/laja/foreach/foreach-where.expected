value = 2
value = 4
